src/proc/align.cpp

```diff
@@ -295,15 +295,29 @@ namespace librealsense
         }
     }
 
+    bool is_special_resolution(const rs2_intrinsics& depth, const rs2_intrinsics& to)
```

Comment:
Providing explanation for the specific case will allow to maintain in future releases